src/propTypes/albumTypes.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+
+export const albumShape = PropTypes.shape({
+  _id: PropTypes.string.isRequired,
+  albumName: PropTypes.string.isRequired,
+  artistName: PropTypes.string.isRequired,
+  image: PropTypes.string.isRequired,
+  // custom validator function takes three arguments
+  releaseDate: (props, propName, componentName) => {
+    if (!/^\d{4}-\d{2}-\d{2}$/.test(props[propName])) {
+      // format YYYY-MM-DD
+      return new Error(
+        `Invalid prop ${propName} supplied to ${componentName}. Validation failed.`
+      );
+    }
+  },
+  averageRating: (props, propName, componentName) => {
+    if (
+      typeof props[propName] !== 'number' ||
+      props[propName] < 1 ||
```

Comment:
According to where our model is defined in the backend, it is possible for averageRating to be 0 if no one has reviewed this album yet: https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/1e48ca722a1a852752cda040cae126beeffd535d/src/models/Album.js#L50

Reply:
<img width="529" alt="screen" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/95896159/e09f305d-338d-421e-8fc8-17783f799cd5">